spike.py

```diff
@@ -0,0 +1,32 @@
+import pygame
+
+
+class Spike:
+
+    def __init__(self, x, y, flip, speed, path_length):
+        self.graphic = pygame.image.load("data/images/Spike.png")
+        if (flip == 1):
+            self.graphic = pygame.transform.flip(self.graphic, True, False)
+        self.rect = self.graphic.get_rect()
+
+        self.initial = y
+
+        self.x = x
```

Comment:
Could you move these close to the function definition as it makes it more readable.